### PR TITLE
Update availability-phrase-resources.js

### PR DESCRIPTION
**Pull request by wxenakis19**
Fixes translation of "Everyday" in japanese

### CODE REVIEW

**Comment by arcaartem** [comment]
👍

File: lib/availability-phrase-resources.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ var otherLocalisedStrings = {
     EveryDay: "Diario"
   },
   ja: {
-    EveryDay: "毎日の"
+    EveryDay: "月〜日（毎日）"
```
Comment (mattiaerre) [inline lib/availability-phrase-resources.js]: I see that there is a space at the very end of the string; did you check w/ Miwa if this is alright?

// cc @arcaartem @wxenakis19

**Comment by mattiaerre** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;4pSn0aL3&#x2F;464-missing-wrong-translations-for-offers"> Missing&#x2F;wrong translations for Offers</a>

**Comment by arcaartem** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;4pSn0aL3&#x2F;464-missing-wrong-translations-for-offers"> Missing&#x2F;wrong translations for Offers</a>

**Pull request merged**